libs/cardano-ledger-api/src/Cardano/Ledger/Api/State/Query.hs

```diff
@@ -19,6 +20,9 @@ module Cardano.Ledger.Api.State.Query (
   -- * @GetDRepState@
   queryDRepState,
 
+  -- * @GetDRepDelegations@
+  queryDRepDelegations,
```

Comment:
Also this addition does deserve a [changelog entry here](https://github.com/IntersectMBO/cardano-ledger/blob/12e4531da33184cbca080a3ddf7cd0196d516665/libs/cardano-ledger-api/CHANGELOG.md?plain=1#L4) (Also no need to add a new section, since 1.12.0.0 version hasn't been released yet)